### ISSUE TITLE
Inaccurate Reverse Geocoding In Europe

### ISSUE THREAD

**Issue by geoops** [pelias/pelias#962]
**Describe the bug**

When using the reverse geocoding feature in Pelias there are inaccurate results in some European regions.

One specific example is lat/long: 51.510333, -3.186567
https://pelias.github.io/compare/#/v1/reverse?point.lat=51.510333&point.lon=-3.186567

Pelias will return the region and county of Derbyshire which is not correct.

```json
{
            "type": "Feature",
            "geometry": {
                "type": "Point",
                "coordinates": [
                    -3.18764,
                    51.510935
                ]
            },
            "properties": {
                "id": "node/7781277404",
                "gid": "openstreetmap:venue:node/7781277404",
                "layer": "venue",
                "source": "openstreetmap",
                "source_id": "node/7781277404",
                "country_code": "GB",
                "name": "Heath Tennis In The Park",
                "street": "King George V Drive",
                "postalcode": "CF14 4EP",
                "confidence": 0.7,
                "distance": 0.1,
                "accuracy": "point",
                "country": "United Kingdom",
                "country_gid": "whosonfirst:country:85633159",
                "country_a": "GBR",
                "macroregion": "England",
                "macroregion_gid": "whosonfirst:macroregion:404227469",
                "region": "Derbyshire",
                "region_gid": "whosonfirst:region:1360698539",
                "county": "Derbyshire",
                "county_gid": "whosonfirst:county:1360698765",
                "county_a": "DBY",
                "localadmin": "Heath and Holmewood",
                "localadmin_gid": "whosonfirst:localadmin:404451883",
                "locality": "Heath and Holmewood",
                "locality_gid": "whosonfirst:locality:1360757427",
                "borough": "Heath",
                "borough_gid": "whosonfirst:borough:1360887587",
                "continent": "Europe",
                "continent_gid": "whosonfirst:continent:102191581",
                "label": "Heath Tennis In The Park, Heath and Holmewood, England, United Kingdom",
                "addendum": {
                    "osm": {
                        "operator": "Lawn Tennis Association",
                        "website": "https://clubspark.lta.org.uk/heathparktennis",
                        "phone": "+44 7713 278 602",
                        "opening_hours": "24/7",
                        "opening_hours:covid19": "open"
                    }
                },
                "index": 0
            }
        }
```
The OSM node is clearly in the city of Cardiff which is about 150mi away.

OSM's lookup tool will place you in Cardiff as well 
https://www.openstreetmap.org/search?query=51.5109349%2C%20-3.1876400

I don't understand why [Derbyshire](https://spelunker.whosonfirst.org/id/1360698765/) is being returned when the lat/long reside within the bbox of [Cardiff](https://spelunker.whosonfirst.org/id/101750397/)

**Steps to Reproduce**

1. Use public Pelias reverse geocode tool: https://pelias.github.io/compare/#/v1/reverse?point.lat=51.510333&point.lon=-3.186567
2. Examine results
3. Compare to other lookup tools
a. https://www.google.com/maps/@51.510333,-3.186567,13z?entry=ttu
b. https://www.openstreetmap.org/search?query=51.5109349%2C%20-3.1876400
4. See the results from Pelias appear to be inaccurate

**Expected behavior**

Using the Pelias reverse geocoding of lat/long 51.510333, -3.186567 should result within the city of Cardiff, UK.

**Environment (please complete the following information):**

Public Pelias API

**Comment by missinglink** [comment]
Hi @geoops, there is currently a massive effort going on to rework a lot of the administrative polygon data we use in the UK.
You can follow along with the thread here: https://github.com/whosonfirst-data/whosonfirst-data/issues/1871#issuecomment-1773546339

I believe that your issue will be resolved once that work is complete.

**Comment by missinglink** [comment]
Do you have examples of other European countries (outside the UK)?

**Comment by missinglink** [comment]
Hmm on closer inspection this is very very wrong 🤔 

<img width="421" alt="Screenshot 2023-11-22 at 17 31 11" src="https://github.com/pelias/pelias/assets/738069/a9ba85e2-023a-4978-a8b6-dcad3f8cc204">

**Comment by missinglink** [comment]
It seems the issue is this record named 'Heath' which is being mistaken for one of the same name in England.

https://spelunker.whosonfirst.org/id/1360887587/

**Comment by missinglink** [comment]
I'm going to close this issue as it's related specifically to data.

A [corresponding issue has been created](https://github.com/whosonfirst-data/whosonfirst-data/issues/2170).
Once that issue is resolved the data updates will be available at https://geocode.earth/data/whosonfirst/ within hours.

**Comment by missinglink** [comment]
If you have further cases please report them in new issue(s)

**Comment by geoops** [comment]
Hey @missinglink was that overhaul ever merged? I can't tell exactly from the thread you [linked](https://github.com/whosonfirst-data/whosonfirst-data/issues/1871#issuecomment-1773546339) but it doesn't appear that it has.

Regarding this specific reverse result, this 'Heath' seems to be correct https://spelunker.whosonfirst.org/id/1360887587/ but if we examine the result from the reverse geocode the region and county are both in [Derbyshire](https://spelunker.whosonfirst.org/id/1360698765/) which is very far away from [Cardiff](https://spelunker.whosonfirst.org/id/1360887587/)

Is this something that the overhaul is/was looking to fix?
